src/fontMetricsData.js

```diff
@@ -484,13 +483,18 @@ export default {
         "126": [0.35, 0.34444, 0, 0, 0.575],
         "168": [0, 0.69444, 0, 0, 0.575],
         "172": [0, 0.44444, 0, 0, 0.76666],
-        "175": [0, 0.59611, 0, 0, 0.575],
         "176": [0, 0.69444, 0, 0, 0.86944],
         "177": [0.13333, 0.63333, 0, 0, 0.89444],
-        "180": [0, 0.69444, 0, 0, 0.575],
+        "198": [0, 0.68611, 0, 0, 1.04166],
```

Comment:
feature request for the future: it would be nice if the script that generates this file included comments up at the top saying that it is an autogenerated file and explaining how to regenerate it.

Reply:
Good call, I've opened an issue for this.